### DISCUSSION TITLE
Web search in GitHub Copilot Chat now available for Copilot Individual

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#142253] [Copilot News and Announcements]
![unnamed](https://github.com/user-attachments/assets/66bbbeac-b07b-4912-ad07-fc6ad33c2a49)

**GitHub Copilot Chat** now supports **web search** in VS Code, Visual Studio, and GitHub.com! 🎉  This means you can effortlessly chat about the latest trends, technologies, and developments right from your coding environment. 

### Why You'll Love It
Imagine asking Copilot about recent events or technologies and getting real-time, relevant answers. Whether you’re curious about the latest release of Node.js or want insights on SAT tokens securing against vulnerabilities, Copilot has your back!

### Getting Started
1. **Enable** the "Copilot Access to Bing" policy in your Copilot Settings.
2. Start using Copilot Chat to ask your burning questions.

For more details, check out our [documentation](https://docs.github.com/en/copilot/using-github-copilot/asking-github-copilot-questions-in-your-ide?tool=vscode#using-github-skills-for-copilot).

### We Want to Hear from You!
How do you plan to use web search with Copilot? Share your thoughts and ideas below! 👇

**Comment by testworksau** [comment]
Well this is disappointing!

I enabled this capability in our GitHub Enterprise, where we have Copilot for Business enabled:

<img width="1314" height="100" alt="image" src="https://github.com/user-attachments/assets/224fcc3b-4a50-43aa-a196-aae5fc7df6d3" />

However Copilot Chat refuses to provide "real-time, relevant answers".
<img width="861" height="1307" alt="image" src="https://github.com/user-attachments/assets/dd47feac-6b5e-479e-af3b-4993cc7b91bb" />

**Reply by testworksau** [reply]
I noticed my Chat extension was outdated in VS Code so I updated it, and restarted. The results are even more outdated now 🙃 

<img width="692" height="286" alt="image" src="https://github.com/user-attachments/assets/71cf7225-8732-4002-a452-88df0b0b621f" />

<img width="845" height="154" alt="image" src="https://github.com/user-attachments/assets/def05f0f-2a50-4715-a83c-15d6a8c41f0d" />

**Reply by Martindeva66** [reply]
Think I am very grateful
